pandas-stubs/_libs/lib.pyi

```diff
@@ -1,3 +1,8 @@
+from enum import Enum
+
 no_default = None
 
+class NoDefault(Enum):
```

Comment:
remove this - it shouldn't be used by users.  It's acting as a sentinel for internal type checking

Reply:
Fixed.